src/components/DevelopmentGuide/Header.mdx

```diff
@@ -1,71 +1,68 @@
-import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
-import { withDesign } from "storybook-addon-designs";
-
-import Header from "./Header";
-import { getCategory } from "../../utils/componentCategories";
-
-<Meta
-  title={getCategory("Header")}
-  component={Header}
-  decorators={[withDesign]}
-  argTypes={{
-    gaOptions: {
-      control: false,
-      table: { defaultValue: { summary: "{}" } },
-    },
-    isProduction: {
-      control: false,
-      table: { defaultValue: { summary: "true" } },
-    },
-  }}
-/>
+import { Meta, Source } from "@storybook/blocks";
 
-# Header
+<Meta title="Development Guide/Header" />
 
-| Component Version | DS Version |
-| ----------------- | ---------- |
-| Added             | `1.1.0`    |
-| Latest            | `1.5.3`    |
+# Header
 
 ## Table of Contents
 
 - [Overview](#overview)
-- [Component Props](#component-props)
+- [Implementation](#implementation)
 - [Accessibility](#accessibility)
 - [Skip Navigation](#skip-navigation)
 - [Catalogs Log In](#catalog-log-in)
 - [Search](#search)
 - [Navigation](#navigation)
 - [Header Log In Timeout](#header-log-in-timeout)
-- [Google Analytics](#google-analytics)
 
 ## Overview
 
-<Description of={Header} />
+The NYPL `Header` is the top most feature on NYPL sites. Even though this component
+appears on various NYPL digital applications, it is itself a separate application
+that gets embedded on said digital applications. This complex component implements
+the Reservoir Design System and contains features for logging in, logging out,
+searching, and navigating the NYPL.org site.
 
-The `Header` component is meant to replicate (and eventually replace) the current
-[NYPL Header package](https://github.com/NYPL/dgx-header-component/) which is used
-across various NYPL digital products, with Reservoir Design System technologies
-and components.
+- [NYPL Header Preview](https://ds-header.nypl.org/header)
+- [NYPL Header App Github repo](https://github.com/NYPL/nypl-header-app)
 
-The `Header` component is a complete and complex component that
+The `Header` component:
 
 - fetches data from different APIs,
 - uses the DS' `SkipNavigation` component for webpage accessibility,
 - has static NYPL.org navigational links,
-- a log in form for patrons to log into NYPL's [Catalog](https://browse.nypl.org/iii/encore/)
+- a log in form for patrons to log into NYPL's [Catalog](https://nypl.na2.iiivega.com/)
   and [Research Catalog](https://www.nypl.org/research/research-catalog/),
 - a search form for patrons to search the Catalog and across [NYPL.org](https://www.nypl.org/).
 
-## Component Props
-
-<Canvas withToolbar>
-  <Story name="Header" args={{ isProduction: true }}>
-    {({ args }) => <Header {...args} />}
-  </Story>
-</Canvas>
-
-<ArgsTable story="Header" />
+## Implementation
+
+The `Header` can be embedded on an NYPL app through the following code snippet:
```

Comment:
Add something to explain why this is important (the fixed height styles).  Also, I think `can` should be changed to `should`.